

  











<!doctype html>
<html>
<head>
    <meta charset="utf-8">
    
        <meta name="viewport" content="width=device-width,initial-scale=1">
    
    
    
    
    
    
    
        
            <title>Sarcoma Dataset Coming Soon to Mapping Cancer Markers Project</title>
        
        
    
    
    
    
    
        


<meta name="Description" content="World Community Grid enables anyone with a computer, smartphone or tablet to donate their unused computing power to advance cutting-edge scientific research on topics related to health, poverty and sustainability."/>
<meta name="Keywords" content="World Community Grid, Volunteer Computing, Let your computer daydream science"/>










<meta property="og:url" content="https://www.worldcommunitygrid.org/about_us/article.s?articleId=571" />
<meta property="og:type" content="website" />
<meta property="og:locale" content="en_US" />
<meta property="og:title" content="Sarcoma Dataset Coming Soon to Mapping Cancer Markers Project" />
<meta property="og:description" content="In this comprehensive update, the Mapping Cancer Markers team explains how they are determining which genes and gene signatures carry the greatest promise for lung cancer diagnosis. They also introduce the next type of cancer--sarcoma--to be added soon to the project. " />
<meta property="og:image" content="https://www.worldcommunitygrid.org/images/logo_images/wcg_logo_square_new_200x200.png" />
<meta property="og:image:secure_url" content="https://www.worldcommunitygrid.org/images/logo_images/wcg_logo_square_new_200x200.png" />
<meta property="og:image:width" content="200" />
<meta property="og:image:height" content="200" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@WCGrid" />
<meta name="twitter:creator" content="@WCGrid" />


    
    
    
    <script>
    var WCG = {};
    WCG.facebook = {};
    WCG.facebook.app_id=112250102148804;
    WCG.locale = {};
    WCG.locale.language='en';
    WCG.locale.country='US';
</script>

    
    
        
<link rel="stylesheet" type="text/css" href="/generated/css/wcg_8ff7f2b046.css" />
<script src="/generated/js/wcg_8a248d7cc7.js"></script>
<script>
	var $j = jQuery.noConflict();
</script>

    
        <link rel="stylesheet" type="text/css" href="/css/lib/jquery-ui-1.11.4/jquery-ui.min.css" />
<link rel="stylesheet" type="text/css" href="/css/lib/jquery-ui-1.11.4/jquery-ui.override.css" />
<script type="text/javascript">
	var $ = $j;
</script>
<script src="/js/lib/jquery-ui-1.11.4/jquery-ui.min.js" ></script>

    
        <script src="/js/lib/jquery.slides.min.js" ></script>
<link rel="stylesheet" type="text/css" href="/css/feature/news-slideshow-styles.css">

    
    
    
    
    
    
    <link href="/static/css/2.7b4bf2a7.chunk.css" rel="stylesheet"/>
<link href="/static/css/main.a189a511.chunk.css" rel="stylesheet"/>

</head>


    <body class="">


    <div class="wcg-legacy-layout wcg-legacy-vertical-flex-container" >
        


    


    

<div id="header" class="header__container" data-context-type="main" data-context-section="" data-context-highlight="News"></div>
        <div id="top" class="wcg-legacy-vertical-flex-item wcg-legacy-vertical-flex-container">
            
            
            <div class="wcg-legacy-vertical-flex-item " >
	            
	            <div class="bx--grid" style="width: 100%">
	                <div class="bx--row">
	                    <div class="bx--col-sm-4 bx--col-md-8 bx--offset-lg-3 bx--col-lg-10 bx--offset-xlg-4 bx--col-xlg-8">

	                    <div class="wcg-legacy wcg-body-padding">
	                        










<main>
    <div id="newsArticleContent" class="news-article">
    <nav class="bx--breadcrumb" aria-label="breadcrumb">
        <div class="bx--breadcrumb-item">
            <a href="/about_us/news.s" class="bx--link" >News</a>
        </div>
    </nav>
    <h1>Sarcoma Dataset Coming Soon to Mapping Cancer Markers Project</h1>
	    <article class="middle-article">
	        
		        <br/>
		        <p>In this comprehensive update, the Mapping Cancer Markers team explains how they are determining which genes and gene signatures carry the greatest promise for lung cancer diagnosis. They also introduce the next type of cancer--sarcoma--to be added soon to the project. </p>
	        
	        <hr/>
	        
			
				<p class="related-items"><span class="related-items-title">Project: </span>
				  
				  <a class="related-items-link" href="/research/mcm1/overview.do">Mapping Cancer Markers</a>
				</p>
			
	        
	            <p class="related-items"><span class="related-items-title">Author: </span>Dr. Igor Jurisica, Krembil Research Institute, University Health Network, Toronto</p>
	        
	        <p class="related-items"><span class="related-items-title">Published on: </span>8 Aug 2018</p>
	        <hr style="margin-top:15px;"/>
	        <section>
	            <span><p>The <a href="https://www.worldcommunitygrid.org/research/mcm1/overview.do" target="_blank">Mapping Cancer Markers</a> (MCM) project continues to process work units for the ovarian cancer dataset. As we accumulate these outcomes, we continue to analyze MCM results from the lung cancer dataset. In this update, we discuss preliminary findings from this analysis. In addition, we introduce the sarcoma dataset that will be our focus in the next stage.</p>

<p><strong>Patterns of gene-family biomarkers in lung cancer</strong></p>

<p>In cancer, and human biology in general, multiple groups of biomarkers (genes, protein, microRNAs, etc.) can have similar patterns of activity and thus clinical utility, helping diagnosis, prognosis or predicting treatment outcome. For each cancer subtype, one could find large number of such groups of biomarkers, each having similar predictive power; yet current statistical and AI-based methods identify only one from a given data set.</p>

<p>We have two primary goals in MCM: 1) to find good groups of biomarkers for the cancers we study, and 2) to identify how and why these biomarkers form useful groups, so we can build a heuristic approach that will find such groups for any disease without needing months of computation on World Community Grid. The first goal will give us not only information that after validation may be useful in clinical practice, but importantly, it will generate data that we will use to validate our heuristics.</p>

<div style="text-align:center">
<figure class="image" style="display:inline-block"><img alt="" height="264" src="/lt/images/mcm1/mcm_aug_1.jpg" width="400" />
<figcaption><br />
Illustration 1: Proteins group by similar interactions and similar biological functions.</figcaption>
</figure>
</div>

<p>Multiple groups of biomarkers exist primarily due to the redundancy and complex wiring of the biological system. For example, the highly interconnected human protein-protein interaction network enables us to see how individual proteins perform diverse molecular functions and together contribute to a specific biological process, as shown above in Illustration 1. Many of these interactions change between healthy and disease states, which in turn affects the functions these proteins carry. Through these analyses, we aim to build models of these processes that in turn could be used to design new therapeutic approaches.</p>

<p>Two specific groups of biomarkers may appear different from each other, yet perform equivalently because the proteins perform similar molecular functions. However, using these groups of biomarkers for patient stratification may not be straightforward. Groups of biomarkers often do not validate in new patient cohorts or when measured by different biological assays, and there are thousands of possible combinations to consider. Some groups of biomarkers may have all reagents available while others may need to be develop (or be more expensive); they may also have different robustness, sensitivity and accuracy, affecting their potential as clinically useful biomarkers.</p>

<p>At the present time, there is no effective approach to find all good groups of biomarkers necessary to achieve the defined goal, such as accurately predicting patient risk or response to treatment.</p>

<p>The first goal of the Mapping Cancer Markers project is to gain a deeper understanding of the &ldquo;rules&rdquo; of why and how proteins interact and can be combined to form a group of biomarkers, which is essential to understanding their role and applicability. Therefore, we are using the unique computational resource of World Community Grid to systematically survey the landscape of useful groups of biomarkers for multiple cancers and purposes (diagnosis and prognosis). Thereby, we established a benchmark for cancer gene biomarker identification and validation. Simultaneously, we are applying unsupervised learning methods such as hierarchical clustering to proteins that group by predictive power and biological function.</p>

<p>The combination of this clustering and the World Community Grid patterns enables us to identify generalized gene clusters that provide deeper insights to the molecular background of cancers, and give rise to more reliable groups of gene biomarkers for cancer detection and prognosis.</p>

<p>Currently, we are focusing on the first-phase results from the lung cancer dataset, which focused on a systematic exploration of the entire space of potential fixed-length groups of biomarkers.</p>

<div style="text-align:center">
<figure class="image" style="display:inline-block"><img alt="" height="499" src="/lt/images/mcm1/mcm_aug_2.jpg" width="400" />
<figcaption><br />
Illustration 2: Workflow of the MCM-gene-pattern-family search. The results of the World Community Grid analysis combined with the unsupervised clustering of genes identifies a set of gene-pattern-families, generalizing the groups of biomarkers. Finally, the results are evaluated using known cancer biomarkers and by using functional annotations, such as signaling pathways, gene ontology function and processes.</figcaption>
</figure>
</div>

<p>As depicted above in Illustration 2, World Community Grid computed about 10 billion randomly selected groups of biomarkers, to help us understand the distribution of which group sizes and biomarker combinations perform well, which in turn we will use to validate heuristic approaches. Analysis showed that about 45 million groups of biomarkers had a high predictive power and passed the quality threshold. This evaluation gives us a detailed and systematic picture of which genes and gene groups carry the most valuable information for lung cancer diagnosis. Adding pathway and protein interaction network data enables us to further interpret and fathom how and why these groups of biomarkers perform well, and what processes and functions these proteins carry.</p>

<p>Simultaneously, we used the described lung cancer data to discover groups of similar genes. We assume that these genes or the encoded proteins fulfill similar biological functions or are involved in the same molecular processes.</p>

<div style="text-align:center">
<figure class="image" style="display:inline-block"><img alt="" height="395" src="/lt/images/mcm1/mcm_aug_3.jpg" width="400" />
<figcaption><br />
Illustration 3: Evaluation of the hierarchical clustering of the lung cancer data, using the complete linkage parameter, for different numbers of groups indicated by the K-values (100 to 1000). The first plot shows the silhouette value - a quality metric in this clustering, i.e., measure of how well each object relates to its cluster compared to other clusters. The second plot depicts the inter- and intra-cluster distance and the ratio of intra/inter cluster distance.</figcaption>
</figure>
</div>

<p>To find the appropriate clustering algorithms and the right number of gene groups (clusters) we use different measures to evaluate the quality of each of the individual clustering. For instance, Illustration 3 (above) shows the results of the evaluation of the hierarchical clustering for different numbers of clusters. To evaluate clustering quality, we used silhouette value (method for assessing consistency within clusters of data, i.e., measure of how well each object relates to its own cluster compared to other clusters). A high silhouette value indicates good clustering configuration, and the figure shows a large increase in the silhouette value at 700 gene groups. Since this indicates a significant increase in quality, we subsequently select this clustering for further analysis.</p>

<p>Not all combinations of biological functions or the lack of it will lead to cancer development and will be biologically important. In the next step, we apply a statistical search to investigate which combinations of clusters are most common among the well-preforming biomarkers, and therefore result in gene groups or pattern families. Since some gene-pattern-families are likely to occur even at random, we use enrichment analysis to ensure the selection only contains families that occur significantly more often than random.</p>

<p>In the subsequent step we validated the selected generalized gene-pattern-families using an independent set of 28 lung cancer data sets. Each of these studies report one or several groups of biomarkers of up- or down-regulated genes that are indicative for lung cancer.</p>

<div style="text-align:center">
<figure class="image" style="display:inline-block"><img alt="" height="178" src="/lt/images/mcm1/mcm_aug_4.jpg" width="400" />
<figcaption><br />
Illustration 4: Shown is a selection of high performing pattern families and how they are supported by 28 previously published gene signatures. Each circle in the figure indicates the strength of the support: The size of the circle represents the number of clusters in the family that where found significantly more often in the signature of this study. The color of the circle indicates the average significance calculated for all clusters in the pattern-family.</figcaption>
</figure>
</div>

<p>Illustration 4 depicts a selection of the most prevalent pattern families and the studies that support them. Each circle in the figure indicates the strength of the support: The size of the circle represents the number of clusters in the family that where found significantly more often in the biomarker of this study. The color of the circle indicates the average significance calculated for all clusters in the pattern-family.</p>

<div style="text-align:center">
<figure class="image" style="display:inline-block"><img alt="" height="122" src="/lt/images/mcm1/mcm_aug_5.jpg" width="400" />
<figcaption><br />
Illustration 5: One of the most frequent gene-pattern-families, is a combination of cluster 1, 7 and 21. We annotated each cluster with pathways using pathDIP and visualized it using word clouds (the larger the word/phrase, the most frequently it occurs).&nbsp;</figcaption>
</figure>
</div>

<p>Finally, we annotated the most effective gene-pattern-families and their gene clusters with molecular functions and pathways that the genes or corresponding proteins are involved in. Illustration 5 shows an example for such a gene-pattern-family that comprises gene-clusters 7, 1 and 21.</p>

<p>The word cloud visualization indicates that cluster 7 is involved in pathways related to GPCRs (G protein&ndash;coupled receptor) and NHRs (nuclear hormone receptors). In contrast, the genes in cluster 1 are highly enriched in EGFR1 (epidermal growth factor receptor) as well as translational regulation pathways. Mutations affecting the expression of EGFR1, a transmembrane protein, have shown to result in different types of cancer, and in particular lung cancer (as we have shown earlier, e.g., (Petschnigg et al., <em>J Mol Biol</em> 2017; Petschnigg et al., <em>Nat Methods</em> 2014)). The aberrations increase the kinase activity of EGFR1, leading to hyperactivation of downstream pro-survival signaling pathways and a subsequent uncontrolled cell division. The discovery of EGFR1 initiated the development of therapeutic approaches against various cancer types including lung cancer. The third group of genes are common targets of microRNAs. Cluster 21 indicates strong involvement with microRNAs, as we and others have shown before (Tokar et al., <em>Oncotarget </em>2018; Becker-Santos et al., <em>J Pathology</em>, 2016; Cinegaglia et al., <em>Oncotarget</em> 2016).</p>

<div style="text-align:center">
<figure class="image" style="display:inline-block"><img alt="" height="173" src="/lt/images/mcm1/mcm_aug_6.jpg" width="400" />
<figcaption><br />
Illustration 6: Evaluation of enriched pathways for cluster 1. Here we used our publicly available pathway enrichment analysis portal pathDIP (Rahmati et al., NAR 2017). The network was generated with our network visualization and analysis tool NAViGaTOR 3 (<a href="http://ophid.utoronto.ca/navigator">http://ophid.utoronto.ca/navigator</a>).</figcaption>
</figure>
</div>

<p>The final illustration evaluates the 20 most significantly enriched pathways for cluster 1. The size of the pathway nodes corresponds to the number of involved genes, and the width of the edges corresponds the number genes of overlapping between pathways. One can see that all pathways involved in translation are highly overlapping. mRNA-related pathways form another highly connected component in the graph. The EGFR1 pathway is strongly overlapping with many of the other pathways, indicating that genes that are affected by those pathways are involved in a similar molecular mechanism.</p>

<p><strong>Sarcoma</strong></p>

<p>After lung and ovarian cancers, next we will focus on sarcoma. Sarcomas are a heterogeneous group of malignant tumors that are relatively rare. They are typically categorized according to the morphology and type of connective tissues that they arise in, including fat, muscle, blood vessels, deep skin tissues, nerves, bones and cartilage, which comprises less than 10% of all malignancies (Jain 2010). Sarcomas can occur anywhere in the human body, from head to foot, can develop in patients of any age including children, and often vary in aggressiveness, even within the same organ or tissue subtype (Honore 2015). This suggests that a histological description by organ and tissue type is neither sufficient for categorization of the disease nor does it help in selecting the most optimal treatment.</p>

<p>Diagnosing sarcomas poses a particular dilemma, not only due to their rarity, but also due to their diversity, with more than 70 histological subtypes, and our insufficient understanding of the molecular characteristics of these subtypes (Jain 2010).</p>

<p>Therefore, recent research studies focused on molecular classifications of sarcomas based on genetic alterations, such as fusion genes or oncogenic mutations. While research achieved major developments in local control/limb salvage, the survival rate for &ldquo;high-risk&rdquo; soft tissue sarcomas (STSs) has not improved significantly, especially in patients with a large, deep, high-grade sarcoma (stage III) (Kane III 2018).</p>

<p>For these reasons, in the next phase of World Community Grid analysis, we will focus on the evaluation of the genomic background of sarcoma. We will utilize different sequencing information and technologies to gain a broader knowledge between the different levels of genetic aberrations and the regulational implications. We will provide a more detailed description of the data and the incentives in the next update.</p>

<ul>
	<li>Petschnigg J, Kotlyar M, Blair L, Jurisica I, Stagljar I, and Ketteler R, Systematic identification of oncogenic EGFR interaction partners, <em>J Mol Biol</em>, <strong>429</strong>(2): 280-294, 2017.</li>
	<li>Petschnigg, J., Groisman, B., Kotlyar, M., Taipale, M., Zheng, Y., Kurat, C., Sayad, A., Sierra, J., Mattiazzi Usaj, M., Snider, J., Nachman, A., Krykbaeva, I., Tsao, M.S., Moffat, J., Pawson, T., Lindquist, S., Jurisica, I., Stagljar, I. Mammalian Membrane Two-Hybrid assay (MaMTH): a novel split-ubiquitin two-hybrid tool for functional investigation of signaling pathways in human cells; <em>Nat Methods</em>, <strong>11</strong>(5):585-92, 2014.</li>
	<li><u>Rahmati, S.</u>, <strong><u>Abovsky, M., Pastrello, C.</u></strong>, <strong>Jurisica, I.</strong> pathDIP: An annotated resource for known and predicted human gene-pathway associations and pathway enrichment analysis. <em>Nucl Acids Res</em>, <strong>45</strong>(D1): D419-D426, 2017.</li>
	<li>Kane, John M., et al. &quot;Correlation of High-Risk Soft Tissue Sarcoma Biomarker Expression Patterns with Outcome following Neoadjuvant Chemoradiation.&quot; Sarcoma 2018 (2018).</li>
	<li>Jain, Shilpa, et al. &quot;Molecular classification of soft tissue sarcomas and its clinical applications.&quot; International journal of clinical and experimental pathology 3.4 (2010): 416.</li>
	<li>Honore, C., et al. &quot;Soft tissue sarcoma in France in 2015: epidemiology, classification and organization of clinical care.&quot; Journal of visceral surgery 152.4 (2015): 223-230.</li>
	<li><strong><u>Tokar T</u></strong>, <strong><u>Pastrello C</u></strong>, Ramnarine VR, Zhu CQ, Craddock KJ, Pikor L, Vucic EA, Vary S, Shepherd FA, Tsao MS, Lam WL<strong>, Jurisica</strong> Differentially expressed microRNAs in lung adenocarcinoma invert effects of copy number aberrations of prognostic genes. <em>Oncotarget</em>. <strong>9</strong>(10):9137-9155, 2018</li>
	<li>Becker-Santos, D.D., Thu, K.L, English, J.C., Pikor, L.A., Chari, R., Lonergan, K.M., Martinez, V.D., Zhang, M., Vucic, E.A., Luk, M.T.Y., Carraro, A., Korbelik, J., Piga, D., Lhomme, N.M., <strong><u>Tsay, M.J.</u></strong>, Yee, J., MacAulay, C.E., Lockwood, W.W., Robinson, W.P., <strong>Jurisica, I.</strong>, Lam, W.L., Developmental transcription factor NFIB is a putative target of oncofetal miRNAs and is associated with tumour aggressiveness in lung adenocarcinoma, <em>J Pathology</em>, <strong>240</strong>(2):161-72, 2016.</li>
	<li><u>Cinegaglia</u><u>, N.C.</u>, Andrade, S.C.S., <u>Tokar, T.</u>, Pinheiro, M., Severino, F. E., Oliveira, R. A., Hasimoto, E. N., Cataneo, D. C., Cataneo, A.J.M., Defaveri, J., Souza, C.P., Marques, M.M.C, Carvalho, R. F., Coutinho, L.L., Gross, J.L., Rogatto, S.R., Lam, W.L., <strong>Jurisica, I.</strong>, Reis, P.P. Integrative transcriptome analysis identifies deregulated microRNA-transcription factor networks in lung, adenocarcinoma, <em>Oncotarget</em>, <strong>7</strong>(20): 28920-34, 2016.</li>
</ul>

<p><strong>Other news</strong></p>

<p>We have secured a major funding from Ontario Government for our research: <em>The Next Generation Signalling Biology Platform</em>. The main goal of the project is developing novel integrated analytical platform and workflow for precision medicine. This project will create an internationally accessible resource that unifies different types of biological data, including personal health information&mdash;unlocking its full potential and making it more usable for research across the health continuum: from genes and proteins to pathways, drugs and humans.</p>

<p>We have also published papers describing several tools, portals and applications with our collaborators. Below we list those most related directly or indirectly to work on World Community Grid:</p>

<ul>
	<li><u>Wong, S.</u>, <strong><u>Pastrello, C., Kotlyar, M.,</u></strong> Faloutsos, C., <strong>Jurisica, I</strong>. SDREGION: Fast spotting of changing communities in biological networks.<em> ACM KDD Proceedings</em>, 2018. In press. <em>BMC Cancer, </em><strong>18</strong>(1):408, 2018.</li>
	<li><strong><u>Kotlyar, M., Pastrello, C., Rossos, A.,</u></strong> <strong>Jurisica, I.</strong> Protein-protein interaction databases. Eds. Cannataro, M. et al. <em>Encyclopedia of Bioinformatics and Computational Biology</em>, <strong>81</strong>, Elsevier. In press. doi.org/10.1016/B978-0-12-811414-8.20495-1</li>
	<li><u>Rahmati, S., <strong>Pastrello, C., Rossos, A.,</strong></u> <strong>Jurisica, I</strong>. Two Decades of Biological Pathway Databases: Results and Challenges, Eds. Cannataro, M. et al. <em>Encyclopedia of Bioinformatics and Computational Biology</em>, <strong>81</strong>, Elsevier. In press.</li>
	<li><u>Hauschild, AC</u>, <strong><u>Pastrello, C., Rossos, A.,</u></strong> <strong>Jurisica, I</strong>. Visualization of Biomedical Networks, Eds. Cannataro, M. et al. <em>Encyclopedia of Bioinformatics and Computational Biology</em>, <strong>81</strong>, Elsevier. In press.</li>
	<li>Sivade Dumousseau M, Alonso-L&oacute;pez D, Ammari M, Bradley G, Campbell NH, Ceol A, Cesareni G, Combe C, De Las Rivas J, Del-Toro N, Heimbach J, Hermjakob H, <strong>Jurisica I</strong>, Koch M, Licata L, Lovering RC, Lynn DJ, Meldal BHM, Micklem G, Panni S, Porras P, Ricard-Blum S, Roechert B, Salwinski L, Shrivastava A, Sullivan J, Thierry-Mieg N, Yehudi Y, Van Roey K, Orchard S. Encompassing new use cases - level 3.0 of the HUPO-PSI format for molecular interactions. <em>BMC Bioinformatics,</em> <strong>19</strong>(1):134, 2018.</li>
	<li>Minatel BC, Martinez VD, Ng KW, Sage AP, <strong><u>Tokar T</u></strong><u>,</u> Marshall EA, Anderson C, Enfield KSS, Stewart GL, Reis PP, <strong>Jurisica I</strong>, Lam WL., Large-scale discovery of previously undetected microRNAs specific to human liver. <em>Hum Genomics,</em> <strong>12</strong>(1):16, 2018.</li>
	<li><strong><u>Tokar T</u></strong>, <strong><u>Pastrello C</u></strong>, Ramnarine VR, Zhu CQ, Craddock KJ, Pikor L, Vucic EA, Vary S, Shepherd FA, Tsao MS, Lam WL<strong>, Jurisica, I.</strong> Differentially expressed microRNAs in lung adenocarcinoma invert effects of copy number aberrations of prognostic genes. <em>Oncotarget</em>. <strong>9</strong>(10):9137-9155, 2018.</li>
	<li>Paulitti A, Corallo D, Andreuzzi E, Bizzotto D, Marastoni S, Pellicani R, Tarticchio G, <strong><u>Pastrello C</u></strong>, <strong>Jurisica I</strong>, Ligresti G, Bucciotti F, Doliana R, Colladel R, Braghetta P, Di Silvestre A, Bressan G, Colombatti A, Bonaldo P, Mongiat M. Matricellular EMILIN2 protein ablation ca 1 uses defective vascularization due to impaired EGFR-dependent IL-8 production, <em>Oncogene</em>, Feb 27. doi: 10.1038/s41388-017-0107-x. [Epub ahead of print] 2018.</li>
	<li><strong><u>Tokar, T.</u></strong><u>,</u> <strong><u>Pastrello, C., Rossos, A., Abovsky, M.,</u></strong> <u>Hauschild, A.C.,</u> <strong><u>Tsay, M., Lu, R.,</u></strong> <strong>Jurisica. I.</strong> mirDIP 4.1 &ndash; Integrative database of human microRNA target predictions, <em>Nucl Acids Res, </em><strong>D1</strong>(46): D360-D370, 2018.</li>
	<li><strong><u>Kotlyar M., Pastrello, C., Rossos, A.,</u></strong><strong> Jurisica, I.</strong>, Prediction of protein-protein interactions, <em>Current Protocols in Bioinf</em>, <strong>60</strong>, 8.2.1&ndash;8.2.14., 2017.</li>
	<li>Singh, M., Venugopal, C., <strong><u>Tokar, T.,</u></strong> Brown, K.B., McFarlane, N., Bakhshinyan, D., Vijayakumar, T., Manoranjan, B., Mahendram, S., Vora, P., Qazi, M., Dhillon, M., Tong, A., Durrer, K., Murty, N., Hallet, R., Hassell, J.A., Kaplan, D., <strong>Jurisica, I.</strong>, Cutz, J-C., Moffat, J., Singh, D.K., RNAi screen identifies essential regulators of human brain metastasis initiating cells,<em> Acta Neuropathologica</em>, <strong>134</strong>(6):923-940, 2017.</li>
</ul>

<p><strong>Thank you</strong></p>

<p>This work would not be possible without the participation of World Community Grid Members. Thank you for generously contributing CPU cycles, and for your interest in this and other World Community Grid projects.</p>
</span>
	        </section>
            
            
            
	        <hr/>
	        
	        
	        
	            <h3>Related Articles</h3>
	            <ul>
	                
					    
					    
					        
					        <li><a href="/about_us/article.s?articleId=443"> One step closer to identifying lung cancer biomarkers</a></li>
					    
	                
					    
					    
					        
					        <li><a href="/about_us/article.s?articleId=457">Working to detect lung and ovarian cancers before they start</a></li>
					    
	                
					    
					    
					        
					        <li><a href="/about_us/article.s?articleId=504">Mapping Cancer Markers Begins Analyzing Lung and Ovarian Cancer Data</a></li>
					    
	                
					    
					    
					        
					        <li><a href="/about_us/article.s?articleId=520">Mapping Cancer Markers Team Analyzes Lung Cancer Data</a></li>
					    
	                
					    
					    
					        
					        <li><a href="/about_us/article.s?articleId=545">New Facility and Expanded Plans for the Mapping Cancer Markers Research Team</a></li>
					    
	                
					    
					    
					        
					        <li><a href="/about_us/article.s?articleId=617">Mapping Cancer Markers Researchers Analyzing Lung Cancer Data</a></li>
					    
	                
	            </ul>
	        
	        
	        
	    </article>
    </div>
</main>

	                    </div>
	                    </div>
	                </div>
	            </div>
            </div>
        </div>
        <div id="footer"></div>
    </div>
    
    <script src="/static/js/runtime-main.eec01e85.js">
</script>
<script src="/static/js/2.2593f09a.chunk.js">
</script>
<script src="/static/js/main.859d1bf2.chunk.js">
</script>

    
    




<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-P7SR87"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-P7SR87');</script>



<script>
wcg_init();
</script>
</body>
</html>
